.changeset/cold-shirts-vanish.md

```diff
@@ -0,0 +1,6 @@
+---
+'@siemens/ix': patch
+---
+
+fix `ix-pill` layout/spacing, adapt tests and add examples to docs
```

Comment:
```suggestion
Adjust `ix-pill` spacing between text + icon and icon only
```

Information about testing und examples is not that relevant for an changelog entry